bitnami/elasticsearch/templates/curator/configmap.yaml

```
@@ -13,7 +13,7 @@ metadata:
   annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
   {{- end }}
 data:
-  action_file.yml: {{ required "A valid .Values.curator.configMaps.action_file_yml entry is required!" (toYaml .Values.curator.configMaps.action_file_yml | indent 2) }}
+  action_file.yml: {{ toYaml (include "common.tplvalues.render" ( dict "value" (required "A valid .Values.curator.configMaps.action_file_yml entry is required!" .Values.curator.configMaps.action_file_yml) "context" $ )) | nindent 2 }}
```

Could you please use `indent 2` instead of `nindent 2`?
```suggestion
  action_file.yml: {{ toYaml (include "common.tplvalues.render" ( dict "value" (required "A valid .Values.curator.configMaps.action_file_yml entry is required!" .Values.curator.configMaps.action_file_yml) "context" $ )) | indent 2 }}
```

```yaml
#Using nindent2
data:
  action_file.yml:
  |-
    ---
    actions:
      1:
        action: delete_indices
        description: "Clean up ES by deleting old indices"
        options:
          timeout_override:
          continue_if_exception: False
```
```yaml
#Using indent 2
data:
  action_file.yml:  |-
    ---
    actions:
      1:
        action: delete_indices
        description: "Clean up ES by deleting old indices"
        options:
          timeout_override:
          continue_if_exception: False
```